src/01-getting-started-in-r/01-intro.Rmd

```diff
@@ -120,8 +120,9 @@ Places to get help include:
 In this chapter we've covered:
 
 -   How to use RStudio to do R programming, both remotely and locally
--   Installing the `tidyverse()` package, the basis of the subsequent code in this book
--   How to customize the appearance of RStudio so you don't burn out your eyes at night
+-   The benefits of working remotely using platforms like the University of Toronto’s        JupyterHub R Studio server.
```

Comment:
There seem to be a bunch of extra spaces here, please remove them (same in the next line as well). Or if you used a tab, delete that and use a space instead.